.travis.yml

```diff
@@ -26,6 +26,7 @@ matrix:
         - PEP8COVERAGE=true    # coverage test are only 
 install:
   - pip3 install --upgrade pip cython numpy || pip3 install --upgrade --user pip cython numpy # all three OSes agree about 'pip3'
+  - pip3 install -r requirements.txt
```

Comment:
What does this?

Reply:
Well, I included that to make sure Travis used the sklearn version specified in requirements.txt. However, I did not change the version used in requirements.txt. I didnt remember that I left that line there, so maybe that is the reason why the CI is working now.

Reply:
Remove it. I fixed the errors in develop.